core/server/master/src/main/java/alluxio/master/file/InodePermissionCheckerFactory.java

```diff
@@ -0,0 +1,17 @@
+package alluxio.master.file;
+
+/**
+ * Factory for {@link InodePermissionChecker}.
+ */
+public final class InodePermissionCheckerFactory {
+  /**
+   * @return a new instance of {@link InodePermissionChecker}
+   */
+  public static InodePermissionChecker create() {
```

Comment:
I think with java 8, you can have a static create method implementation in an interface, so this functionality could probably go into the `InodePermissionChecker` interface definition.

Reply:
done